plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/components/FastXSLTMediatorPropertiesEditionComponent.java

```diff
@@ -260,9 +261,10 @@ public void updatePart(Notification msg) {
      * 
      */
     public String getHelpContent(Object key, int kind) {
-        if (key == EsbViewsRepository.FastXSLTMediator.Basic.fastXsltStaticSchemaKey)
-            return ""; //$NON-NLS-1$
-        return super.getHelpContent(key, kind);
+//        if (key == EsbViewsRepository.FastXSLTMediator.Basic.fastXsltStaticSchemaKey)
```

Comment:
Remove commented lines.

Reply:
Fixed in https://github.com/wso2/devstudio-tooling-esb/pull/806